### PR TITLE
Dockerize and build this a8b

### PR DESCRIPTION
**Pull request by juancarrey**
Rename stuff to get go to build properly.
Build on Docker
Create builder image (runnable for now..)

### CODE REVIEW

**Comment by javierlopezdeancos** [comment]
Looks good to ME!

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+ADD . /go/src/ask8ball
+WORKDIR /go/src/ask8ball
+RUN dep ensure
+RUN go build
+RUN go install
+ENTRYPOINT ["/go/bin/ask8ball"]
+
+#FROM scratch
+#COPY --from=builder /go/bin/ask8ball /go/bin/ask8ball
+#ENTRYPOINT ["/go/bin/ask8ball"]
```
Comment (vicpada) [inline Dockerfile]: Remove comments

File: main.go [hunk 2]
```diff
@@ -109,8 +111,8 @@ func parseGroups(api *slack.Client) (map[string]bool, error) {
 		}
 	}
 
-	allowed["GGYT2NJA1"] = true;
-	allowed["CGYT18RMK"] = true;
+	allowed["GGYT2NJA1"] = true
+	allowed["CGYT18RMK"] = true
```
Comment (vicpada) [inline main.go]: What are these hardcoded values ?

Comment (juancarrey) [reply main.go]: The two groups of our slack in which ask8ball can answer. The channels where it got invited was not working and I had not much time so I just hardcoded it back then, we need to refactor this and make it properly.

**Pull request merged**